shop/src/components/SentryErrorBoundary.js

```diff
@@ -19,33 +21,58 @@ class ErrorBoundary extends React.Component {
   }
 }
 
+const isAdminPage = (locationHash) =>
+  /^#\/(super-)?admin(\/.*)?$/.test(locationHash)
+
 const SentryErrorBoundary = ({ children }) => {
   const { enabled: sentryEnabled, sentry } = useSentry()
+  const { config } = useConfig()
 
   const [errorEvent, setErrorEvent] = useState()
+  const [sentryInitialized, setSentryInitialized] = useState(false)
+
+  const shouldLogUserErrors = _get(config, 'logErrors', false)
 
   const eventProcessor = (event) => {
-    if (event.skipProcessing) return _omit(event, ['skipProcessing'])
+    if (!event.skipProcessing) {
+      const pathname = new URL(event.request.url)
+      const adminPage = isAdminPage(pathname.hash)
+
+      if (!adminPage) {
+        if (!window.shouldLogUserErrors) {
+          console.error('Exception occurred', event)
+          return null
+        }
+
+        // Ask permission from the user, if it is not in admin page
+        setErrorEvent(event)
+        return null
+      }
+    }
 
-    const pathname = new URL(event.request.url)
-    const isAdminPage = /^#\/(super-)?admin(\/.*)?$/.test(pathname.hash)
-    if (!isAdminPage) {
-      // Ask permission from the user, if it is not in admin page
-      setErrorEvent(event)
+    if (process.env.NODE_ENV === 'development') {
```

Comment:
When running on local is NODE_ENV always set to 'development"? Even like in watch mode? Just asking to be sure.

Reply:
Thanks. Changing this to  `!== production`